dev/make-distribution.sh

```diff
@@ -150,7 +156,13 @@ export MAVEN_OPTS="${MAVEN_OPTS:--Xmx2g -XX:MaxPermSize=512M -XX:ReservedCodeCac
 # Store the command as an array because $MVN variable might have spaces in it.
 # Normal quoting tricks don't work.
 # See: http://mywiki.wooledge.org/BashFAQ/050
-BUILD_COMMAND=("$MVN" clean package -DskipTests $@)
+
+# If NUM_THREADS is set we actually want to add the -T in (removed with shift earlier)
+if [ -n "$NUM_THREADS" ]; then
+  MVN_T_OPTION="-T"
+fi
+
+BUILD_COMMAND=("$MVN" $MVN_T_OPTION $NUM_THREADS clean package -DskipTests $@)
```

Comment:
That's true and it's unlikely users will use anything but 1C as the value, so I agree let's add -T 1C and keep it simple

Changeset will therefore be

- Adding the comment in README.md that you can use it
- Hardcodeing the above
- Adding the missing hadoop-2.7 example profile